contrib/kafka/pkg/reconciler/resources/receive_adapter.go

```diff
@@ -100,4 +100,9 @@ func MakeReceiveAdapter(args *ReceiveAdapterArgs) *v1.Deployment {
 			},
 		},
 	}
+	if args.ImagePullPolicy != nil {
+		d.Spec.Template.Spec.Containers[0].ImagePullPolicy = *args.ImagePullPolicy
+	}
```

Comment:
If image pull policy is configured in controller yaml then it is set for receiver adapter